client/forms.py

```diff
@@ -20,19 +19,20 @@
 data = json.load(json_file)
 onlyNames = [a["name"] for a in data]
 onlyNames.sort()
-CITIES = [(str(x),str(x)) for x in onlyNames]
+CITIES = [(str(x), str(x)) for x in onlyNames]
```

Comment:
Auto PEP8 :)

Reply:
I meant why generating a tuple like that, anyway since this is not a code that was changed right now we are ignoring this (unfortunately...)
we will refactor stuff later.